dashboard/generate_subnational_map.R

```diff
@@ -7,71 +7,86 @@ library(sf)
 library(tmap)
 
 # read in data to display
-ipc_indices_data <- read.csv("Nov2020_FewsNetGlobalIPC.csv")
+ipc_indices_data <- read.csv("data/foodinsecurity/ethiopia_foodinsec_trigger.csv") #Nov2020_FewsNetGlobalIPC.csv")
```

Comment:
should this csv now be for all countries? does the new dashboard need a specific format?

Reply:
No, the csv only includes Ethiopia at the moment. However, the script to compute the numbers is easily transferrable to other countries, but would need few changes on the dashboard side to built-in that flexibility. 
Added few details about required data file format